amazon-kinesis-client/src/main/java/software/amazon/kinesis/leases/dynamodb/DynamoDBLeaseTaker.java

```diff
@@ -71,6 +73,8 @@ public DynamoDBLeaseTaker(LeaseRefresher leaseRefresher, String workerIdentifier
             final MetricsFactory metricsFactory) {
         this.leaseRefresher = leaseRefresher;
         this.workerIdentifier = workerIdentifier;
+        this.leaseDurationMillis = leaseDurationMillis;
+        this.leaseRenewalIntervalMillis = leaseDurationMillis / 3;
```

Comment:
1. We dont need leaseDurationMillis field variable
2. The division by 3, I believe you took it from leaserenewer. Can we expose a method in some common file or LeaseRenewer to reuse it in multiple places.